react/features/recording/components/Recording/LocalRecordingManager.web.ts

```diff
@@ -231,6 +231,11 @@ const LocalRecordingManager: ILocalRecordingManager = {
                 });
             }
 
+            // handle no mic permission
+            if (!getLocalTrack(getTrackState(getState()), MEDIA_TYPE.AUDIO)?.jitsiTrack?.track) {
```

Comment:
Why can't we work without that track?

Reply:
Oh FML. Can we create a fake track with the WebAudio API playing silence perhaps? The current behavior is quite unexpected...

Reply:
Hmm, probably. Not sure tho. We'd need some time for this